build_tools/fetch_sources.py

```diff
@@ -90,7 +96,19 @@ def apply_patches(args):
         patch_files = list(patch_project_dir.glob("*.patch"))
         patch_files.sort()
         log(f"Applying {len(patch_files)} patches")
-        exec(["git", "am", "--whitespace=nowarn"] + patch_files, cwd=project_dir)
+        exec(
+            [
+                "git",
```

Comment:
There are a couple of lines in every workflow that configure the git name/email. Can you remove those in this pr so the CI marches end user usage?